spec/parameters.yaml

```diff
@@ -35,6 +35,36 @@ WebhookPostRequest:
       - url
       - sign
       - on
+FeatureFlagCopyBody:
+  name: featureFlagCopyBody
+  in: body
+  required: true
+  description: Copy feature flag configurations between environments.
+  schema:
+    type: object
+    properties:
+      source:
+        $ref: '#/definitions/FeatureFlagCopyObject'
+      target:
+        $ref: '#/definitions/FeatureFlagCopyObject'
+      comment:
+        type: string
+        description: comment will be included in audit log item for change.
+        example: "This is a comment string"
+      includedActions:
+        type: array
+        items:
+          type: string
+          example: []
+          enum: [updateOn, updatePrerequisites, updateTargets, updateRules, updateFallthrough, updateOffVariation]
+        description: Define the parts of the flag configuration that will be copied.
+      excludedActions:
+        type: array
+        items:
+          type: string
+          example: []
+          enum: [updateOn, updatePrerequisites, updateTargets, updateRules, updateFallthrough, updateOffVariation]
```

Comment:
Maybe we can dry this up into single $ref.

Reply:
latest commit cleans this up